src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -1168,6 +1176,8 @@ private function step_in_head(): bool {
 
 			/*
 			 * > A start tag whose tag name is "script"
+			 *
+			 * @todo Could the adjusted insertion location be anything other than the current location?
```

Comment:
[In this case, html5ever does account for it, but I still don't see how it behaves differently. ](https://github.com/servo/html5ever/blob/fd246b368e4dc9e6ae241051de45dd535af8d76e/html5ever/src/tree_builder/rules.rs#L128-L138)

I don't think this should block the PR, but I do want to follow-up.